gammapy/spectrum/sherpa_utils.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, fit):
     def _calc(self, data, model, *args, **kwargs):
         self.fit.calc_statval()
         # Sum likelihood over all observations
-        total_stat = np.sum(self.fit.statval, dtype=np.float64)
+        total_stat = np.sum([np.sum(v) for v in self.fit.statval], dtype=np.float64)
```

Comment:
* Statval sum does not work for more than one observation

Consider the following code example

```
In [7]: array1 = np.array([1,2,3])

In [8]: array2 = np.array([1,2,3])

In [9]: listofarrays = [array1, array2]

In [10]: np.sum(listofarrays)
Out[10]: 12
```

So ``np.sum`` should be able to deal with a list of arrays. Do you have an example where you got an wrong fit for more than one observation?